0007-onion-routing-over-libp2p.adoc

```diff
@@ -0,0 +1,126 @@
+= Improving privacy through onion routing over libp2p
+Daniel Karzel <daniel.karzel@coblox.tech>;
+:toc:
+:revdate: 2019-06-17
+
+NOTE: Author: {authors} +
+Date: {revdate} +
+Tracking issue: https://github.com/coblox/spikes/issues/8[Tor Support for COMIT]
+
+== Context
+
+Onion routing as implemented by the https://en.wikipedia.org/wiki/Tor_(anonymity_network)[Tor network] enables anonymous usage of the internet
+through end2end multi-time-encryption over multiple hops provided through relay nodes. Anybody in the network can set up a relay node in the network.
```

Comment:
🔧Please follow a one-sentence-per-line policy. This makes the reviewing process easier as one can suggest changes per sentence.